mantidimaging/core/data/dataset.py

```diff
@@ -69,7 +73,7 @@ def __init__(self, stacks: List[Images] = [], name=""):
 
     @property
     def all(self) -> List[Images]:
-        all_images = self._stacks + self.recons
+        all_images = self._stacks + self.recons.data
```

Comment:
It took me a little while to figure out the `.data` because in `Images` it is used to access the array of pixels. It would be good to change the name, see comment below in `ReconList`